pkg/cmd/describe_platform.go

```diff
@@ -60,14 +60,14 @@ type describePlatformCommandOptions struct {
 	*RootCmdOptions
 }
 
-func (command *describePlatformCommandOptions) validate(args []string) error {
+func (command *describePlatformCommandOptions) validate(_ *cobra.Command, args []string) error {
```

Comment:
Any reason why we are changing the method? I mean, we don't use that input variable after all. I guess it's to have the same structure in all method, but it's something that is not really required.

Reply:
I wanted to add it for the consistency, other command have it written the same way so I wanted to be consistent.